rollup/internal/controller/relayer/l1_relayer.go

```diff
@@ -251,14 +251,14 @@ func (r *Layer1Relayer) shouldUpdateGasOracle(baseFee uint64, blobBaseFee uint64
 	}
 
 	expectedBaseFeeDelta := r.lastBaseFee*r.gasPriceDiff/gasPriceDiffPrecision + 1
-	if baseFee >= r.minGasPrice && (baseFee >= r.lastBaseFee+expectedBaseFeeDelta || baseFee+expectedBaseFeeDelta <= r.lastBaseFee) {
+	if baseFee >= r.minGasPrice && math.Abs(float64(baseFee-r.lastBaseFee)) >= float64(expectedBaseFeeDelta) {
```

Comment:
Would it cause a problem to minus two `uint64` first then cast it to `float64`?
for clarity I would suggest casting them first and minus the value, this would not cause misunderstanding.

Reply:
make sense